File: Service/Billing/RMA.Service.Billing.Contracts/Entities/PolicyPaymentAllocation/TransferPaymentFromPolicyToPolicyRequest.cs
using System;

namespace RMA.Service.Billing.Contracts.Entities.PolicyPaymentAllocation
{
    public class TransferPaymentFromPolicyToPolicyRequest
    {
        public int FromPaymentAllocationId { get; set; }
        public int FromPolicyId { get; set; }
        public DateTime FromPolicyBillingMonth { get; set; }
        public int ToPolicyId { get; set; }
        public DateTime ToPolicyBillingMonth { get; set; }
        public decimal AmountToTransfer { get; set; }
        public int RolePlayerId { get; set; }
        public string Notes { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/transfer-payment-from-policy-to-policy-request.ts
export class TransferPaymentFromPolicyToPolicyRequest
{
 fromPolicyId: number;
 toPolicyId: number;
 fromPolicyBillingMonth: Date;
 toPolicyBillingMonth: Date;
 amountToTransfer: number;
 rolePlayerId: number;
 notes: string;
 fromPaymentAllocationId: number;

}